src/jit/bitsetasuint64inclass.h

```diff
@@ -295,16 +295,15 @@ class BitSetUint64Iter
     {
     }
 
-    bool NextElem(unsigned* pElem)
+    int NextElem()
     {
         static const unsigned UINT64_SIZE = 64;
 
         if ((m_bits & 0x1) != 0)
         {
-            *pElem = m_bitNum;
             m_bitNum++;
             m_bits >>= 1;
-            return true;
+            return m_bitNum++;
         }
```

Comment:
This should be just `return m_bitNum`. Currently, m_bitNum is getting incremented twice. I prefer the increment before the `return`, and not using `++` in the `return` clause.